nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/autodiff/samediff/SameDiff.java

```diff
@@ -2669,8 +2738,8 @@ public INDArray outputSingle(Map<String, INDArray> placeholders, String output)
         Preconditions.checkState(outputs != null && outputs.length > 0, "No outputs were specified");
         long threadId = Thread.currentThread().getId();
         if (!sessions.containsKey(threadId)) {
-            log.info("Creating new InferenceSession for thread {}", threadId);
-            sessions.put(threadId, new InferenceSession(this));
+            log.info("Creating getInferenceFactory().create for thread {}", threadId);
```

Comment:
English here doesn't make sense. I would still leave this as is or 
maybe you can do:
```
log.info( "Using {}  creating InferenceSession for thread {}" ,getInferenceFactory().getClass().getName(), Thread.currentThread.getId());
```

This will indicate the thread id and the inference session factory type used.

Reply:
I see. it seems I mistakenly replaced comment while using ctrl+h/replace.  its better to use the old message